contentcuration/contentcuration/static/js/edit_channel/exercise_creation/views.js

```diff
@@ -547,8 +547,12 @@ var EditorView = BaseViews.BaseView.extend({
                     paragraphs[paragraphs.length - 1].innerHTML += '&nbsp;';
                     updatedHtml = div.innerHTML;
                 }
-                self.editor.setHTML(updatedHtml)
+                self.editor.setHTML(updatedHtml);
+                var isBlank = !self.model.get(self.edit_key);
                 self.model.set(self.edit_key, self.model.get(self.edit_key) + formula);
+                if(isBlank) { // Formulas added to blank content disable text selection (needs to be wrapped in spaces)
```

Comment:
There was a problem with the exercise editor if you haven't typed anything and start by adding a formula. Any mathjax strings get converted into svgs to prevent users from messing with the formulas, but when it's the only thing in the editor, there is no way to start editing the actual text. This re-renders the editor to add the extra padding to allow text selection.